ci_framework/roles/edpm_build_images/tasks/install.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Install required packages
+  become: true
+  ansible.builtin.package:
+    name: "{{ cifmw_build_host_packages }}"
+    state: latest
+
+- name: Ensure directories are present
+  ansible.builtin.file:
+    path: "{{ cifmw_edpm_build_images_basedir }}/{{ item }}"
+    state: directory
+  loop:
+    - tmp
+    - artifacts/edpm_images
+
+- name: Check edpm-image-builder repository exist
+  register: eib_stats
+  ansible.builtin.stat:
+    path: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/edpm-image-builder"
+
+- name: Get edpm-image-builder repository
```

Comment:
We don't even need to clone this repo.. For  non check jobs cases - we can just install edpm-image-builder rpm

Reply:
done